worker/src/dns.rs

```diff
@@ -77,6 +77,7 @@ pub fn build_dns_vc(pk: &JWK, domain: String) -> Result<Credential> {
                           "@id": "https://tzprofiles.com/timestamp",
                           "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
                       },
+                      "dnsServerUrl": "https://cloudflare-dns.com/dns-query",
```

Comment:
That URL needs to describe what the field is, then in the evidence map you can fill it with the actual DNS used.

And I'm tempted to put `1.1.1.1` instead of that API, but it might be too deceiving.